admin_manual/installation/source_installation.rst

```diff
@@ -277,6 +277,32 @@ Keep in mind that changes to php.ini may have to be done on more than one ini fi
 
   /etc/php5/cli/php.ini
 
+**Notes when using php-fpm:**
+
+It may be the case when using ``php-fpm``, system environment variables like PATH, TMP or others are not automatically 
+populated in the same way compared when using ``php-cli``.
+
+A php call like ``getenv('PATH');`` can therefore return an empty result.
+Even fallback content for those variables is most times defined, this may not match 100% of the needs. 
+In those cases, the variables need to be set manually in the corresponding ``php-fpm`` ini file. 
+For Ubuntu, the location for these ini files can be found at: ``/etc/php5/fpm/``
+
+Usually, you will find some or all of this environment variables beeing commented out like:
+::
+
+	;env[HOSTNAME] = $HOSTNAME
+	;env[PATH] = /usr/local/bin:/usr/bin:/bin
+	;env[TMP] = /tmp
+	;env[TMPDIR] = /tmp
+	;env[TEMP] = /tmp
+
+You can either just uncomment the existing one and/or copy the output from the command shell eg: ``printenv PATH`` to 
+the environment variable to be set. If no system environment variable is present to be uncommented, 
+you need to create one.
+
+Please keep in mind, that it is possible to create different settings for ``php-cli`` and ``php-fpm``
+and even for different domains/websites. Please take care to take the right one. Your result can be tested with ``phpinfo()``
+which must accessed via a browser to your domain/website.
```

Comment:
Well, the markdown parser won't create a new line unless you leave a space in between paragraphs, but it doesn't mean that it's wrong in the original text. I'll let @carlaschroder decide what's right here.

And an admin should know which files he needs to modify in order to configure PHP for his domain or he's in big trouble when something goes wrong. 
Maybe something like this:
`, so make sure you modify the files which apply to the PHP-FPM pool your ownCloud installation belongs to.`